src/com/ibm/ServerWizard2/ServerWizard2.java

```diff
@@ -75,28 +102,53 @@ public static void main(String[] args) {
 
 		LOGGER.config("======================================================================");
 		LOGGER.config("ServerWiz2 Version "+getVersionString()+" Starting...");
+
+		if(updateOnlyMode) {
+			LOGGER.config("Running in update only mode. GUI would not be loaded.");
+			getPreferencesForUpdateMode();
+		} else {
+			getPreferences();
+		}
 	
-		getPreferences();
-		
 		TargetWizardController tc = new TargetWizardController();
 		SystemModel systemModel = new SystemModel();
-	    MainDialog view = new MainDialog(null);
-	    tc.setView(view);
-	    tc.setModel(systemModel);
-	    view.setController(tc);
-	    tc.init();
-	    systemModel.cleanupMode = cleanupMode;
+		MainDialog view = new MainDialog(null);
+		tc.setView(view);
+		tc.setModel(systemModel);
+		view.setController(tc);
+		tc.init();
+		systemModel.cleanupMode = cleanupMode;
 		if (!inputFilename.isEmpty()) {
 			view.mrwFilename=inputFilename;
 		}
-	    view.open();
+		if(updateOnlyMode) {
+			tc.readXML(inputFilename);
+			if(tc.readXMLfailed) {
+				System.exit(1);
+			}
+			tc.writeXML(outputFilename);
+		} else {
+			view.open();
+		}
 	}
-	
+
+	// Set the GIT location to current working directory
+	private static void getPreferencesForUpdateMode() {
+		try {
+		ServerWizard2.GIT_LOCATION = new java.io.File( "." ).getCanonicalPath();
```

Comment:
How does this work, exactly? You need to invoke the "update" mode from the directory that contains the new attribute and target xml files? Can this be made to take another parameter to point to the location of those files?

Reply:
No, it just looks for "common-mrw-xml" in the current directory. If it is there, it takes it, otherwise it clones the repo from github.
I can make it to take this using a new parameter.

Reply:
I have changed the usage text as per your suggestion.